### PR TITLE
Feature/frontend refactors global styles

### PR DESCRIPTION
**Pull request by usersadvocate**

### CODE REVIEW

File: frontend/src/app/modules/notification/notification.module.ts [hunk 1]
```diff
@@ -1,49 +1,10 @@
...
-    easing: 'ease'
-  },
-  overlap: 150
-}
-};
+import { customNotifierOptions } from '../../shared/constants';
 @NgModule({
   imports: [
-    NotifierModule.withConfig(customNotifierOptions)
+    NotifierModule.withConfig(customNotifierOptions as NotifierOptions)
```
Comment (MatvienkoDO) [inline frontend/src/app/modules/notification/notification.module.ts]: don't think it is good decision to force variable to some type
https://palantir.github.io/tslint/rules/no-object-literal-type-assertion/

File: frontend/src/app/shared/components/list-select/list-select.component.ts [hunk 2]
```diff
@@ -26,13 +26,13 @@ export class ListSelectComponent implements OnInit, OnChanges {
 
   public readonly selected$ = new BehaviorSubject<string[]>([]);
 
-   private panelOpenState: boolean = false;
+   private panelOpenState = false;
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/list-select/list-select.component.ts]: redundant space

File: frontend/src/app/shared/services/account-modal/account-modal.service.ts [hunk 3]
```diff
@@ -29,7 +29,7 @@ export class AccountModalService {
   public openLoginSignup() {
     const dialogRef = this.dialog.open(AccountComponent, {
       width: '550px',
-      height: '590px',
+      height: 'auto',
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/services/account-modal/account-modal.service.ts]: maybe just delete it

File: frontend/src/app/shared/services/notification/notification.service.ts [hunk 4]
```diff
@@ -6,21 +6,21 @@ import { NotifierService } from 'angular-notifier';
   providedIn: 'root'
 })
 export class NotificationService {
-  constructor(private _service: NotifierService ) { }
+  constructor(private service: NotifierService ) { }
 
   success(msg: string) {
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/services/notification/notification.service.ts]: i think 'msg' is not good choice for name. Usage of vowels is not forbidden for us

**Review by MatvienkoDO — CHANGES_REQUESTED** [review]
comments

File: frontend/src/app/shared/services/notification/notification.service.ts (hunk 4 above)
Comment (usersadvocate) [reply frontend/src/app/shared/services/notification/notification.service.ts]: lol

File: frontend/src/app/modules/notification/notification.module.ts [hunk 5]
```diff
@@ -1,49 +1,10 @@
...
-    easing: 'ease'
-  },
-  overlap: 150
-}
-};
+import { customNotifierOptions } from '../../shared/constants';
 @NgModule({
   imports: [
-    NotifierModule.withConfig(customNotifierOptions)
+    NotifierModule.withConfig(customNotifierOptions as any as NotifierOptions)
```
Comment (MatvienkoDO) [inline frontend/src/app/modules/notification/notification.module.ts]: we should not make type casts

File: frontend/src/app/pages/product-details/product-details.component.ts [hunk 6]
```diff
@@ -18,7 +18,7 @@ import {
   selector: 'app-product-details',
   templateUrl: './product-details.component.html',
   styleUrls: ['./product-details.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  changeDetection: ChangeDetectionStrategy.Default,
```
Comment (MatvienkoDO) [inline frontend/src/app/pages/product-details/product-details.component.ts]: just delete

File: frontend/src/app/shared/components/best-sales/best-sales.component.ts [hunk 7]
```diff
@@ -9,7 +9,7 @@ import { Product, CardProduct, ProductResponse } from 'src/app/shared/models';
   selector: 'app-best-sales',
   templateUrl: './best-sales.component.html',
   styleUrls: ['./best-sales.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  changeDetection: ChangeDetectionStrategy.Default,
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/best-sales/best-sales.component.ts]: just delete

File: frontend/src/app/shared/components/favourite-button/favourite-button.component.ts [hunk 8]
```diff
@@ -13,19 +11,8 @@ export class FavouriteButtonComponent implements OnInit {
...
-      // const id = cardProduct.id;
-      // const list = this.favouritesService.getListOfFavourites()
-      // for (let i =0 ; i<list.length; i++){
-      //   if (list[i].id === id){
-      //     this.selected = true;
-      //   }
-      // }
-     }
-     
+  constructor() { }
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/favourite-button/favourite-button.component.ts:13]: delete it

File: frontend/src/app/shared/components/favourite-button/favourite-button.component.ts [hunk 9]
```diff
@@ -13,19 +11,8 @@ export class FavouriteButtonComponent implements OnInit {
...
+
   ngOnInit() {
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/components/favourite-button/favourite-button.component.ts:15]: and it

File: frontend/src/app/shared/constants.ts [hunk 10]
```diff
@@ -7,3 +7,44 @@ export const signupInvalidForm = 'signup_invalid_form';
 export const emailIsNotUnique = 'email_is_not_unique';
 export const incorrectLoginPasswordPair = 'incorrect_login_password_pair';
 export const invalidEmailPassword = 'Invalid email/password';
+
+export const customNotifierOptions = {
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/constants.ts]: should be typed

File: frontend/src/app/shared/styles/_mobile-filters.scss [hunk 11]
```diff
@@ -88,7 +90,7 @@
     top: 0px;
     left: 0;
     z-index: 7;
-    height: 0;
+    height: auto;
```
Comment (MatvienkoDO) [inline frontend/src/app/shared/styles/_mobile-filters.scss]: delete it

**Review by MatvienkoDO — APPROVED** [review]

**Pull request merged**